### ISSUE TITLE
Accessibility: MAS40B: Inspect: 'IsExpandCollapsePatternAvailable' property is FALSE for Git or Extensions or Tasks combo box in Output Panel.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17175]
VS Code Version:1.6.1
OS Version:1607

**Steps to Reproduce:**
1.Open Visual studio code editor.
2. Open any file and using "Ctrl+1" key  navigate to the first vs code editor.
3. Using F1 key navigate to the palette, type "git branch" and "refactor-to-promise" as the branch name
4. Make changes to the code by refactoring, adding/deleting new lines
5. Compare the changes between the HEAD and current file by navigating to View Bar ---> Git ; Navigate to changes and hit ENTER to activate "Diff View" 
6. Stage and commit the changes under View Bar ---> Git  ; Navigate to the "Commit Staged"  button and hit ENTER to make a commit
7. Undo the commit by navigating to the toggle button (same row as commit staged), hit ENTER and use down ARROW to choose "Undo Last Commit" option 
8. Navigate to "More Button" in Git Side Bar and Hit Enter.
9. Navigate to "Show Git Output" and Hit Enter.
10. Navigate to "Git or Extensions or Tasks" combo box in output panel and observe 'IsExpandCollapsePatternAvailable' property in Inspect.

**Expected Result:**
'IsExpandCollapsePatternAvailable' property should be TRUE for Git or Extensions or Tasks combo box in Output Panel.
Narrator Behavior:Narrator should read as "Git editable combo box Git collapsed".

**Actual Result:**
'IsExpandCollapsePatternAvailable' property is FALSE for Git or Extensions or Tasks combo box in Output Panel
Narrator Behavior:
Narrator reads as "Git Git, editable combo box"
NVDA Behavior:
NVDA reads as "Git combo box Git collapsed"

**User Impact:**
Users relying on screen readers will not know if the toolbar is expanded or collapsed.

![isexpandcollapsepatternavailable property is false for git or extensions or tasks combo box in output panel](https://cloud.githubusercontent.com/assets/22956130/21184697/a110dbb2-c233-11e6-9d9d-6b3147122e11.jpg)

**Comment by isidorn** [comment]
@chrisdias @kieferrm do we now need to support Narator?
@WiproPSAccTest As far as I see from this bug description we are working as expected on NVDA but not for Narator. The problem is that narator does not properly read out the native HTML select box, and NVDA does it nicely.
I can add some workarounds for the narator bug on my side but would prefer not to do anything if we are not supporting Narator

I think we have bigger issues in the editor with narator

**Comment by isidorn** [comment]
Closing as this issue is only appearing on narator. Correct me if I am wrong here

**Comment by WiproPSAccTest** [comment]
As per the comments narrator was not part of the scope.

**Comment by chrisdias** [comment]
Re-opening per WiproPSAccTest verification.

The problem i see here is that there is no way to easily navigate the combo box in the panel.  If i press down arrow it moves to the next item in the list and then focus jumps to the pane.  I can't find a way with the keyboard to drop down the combo box.  

how do i make the drop down open with the keyboard?

![image](https://user-images.githubusercontent.com/1487073/27667184-e2237c06-5c2d-11e7-9a39-b2b4a0c80ae3.png)

**Comment by isidorn** [comment]
I agree, we have an issue here, however not a regression so assigning to July.

**Comment by isidorn** [comment]
Now it is possible to expand the select box with the space key which is also how select boxes natively behave.
On the mac the keydown also nicely expands it, not sure why this is not working on windows
